src/Microsoft.Extensions.Configuration.AzureKeyVault/project.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Package needs to be added to `NuGetPackageVerifier.json` + Coherence\packages\packages.csv